packages/hardhat/contracts/periphery/lib/ControllerHelperUtil.sol

```diff
@@ -93,21 +93,25 @@ library ControllerHelperUtil {
      */
     function mintAndLp(address _controller, address _nonfungiblePositionManager, address _wPowerPerp, address _wPowerPerpPool, address _weth, ControllerHelperDataType.MintAndLpParams calldata _mintAndLpParams, bool _isWethToken0) public returns (uint256, uint256) {
         IWETH9(_weth).withdraw(_mintAndLpParams.collateralToDeposit);
-
+        console.log('withdrawn collateral looking at getAmounttoLP with %s', _mintAndLpParams.collateralToLp);
         (uint256 amount0Desired, uint256 amount1Desired) = getAmountsToLp(_wPowerPerpPool, _mintAndLpParams.collateralToLp, _mintAndLpParams.wPowerPerpAmount, _mintAndLpParams.lowerTick, _mintAndLpParams.upperTick, _isWethToken0);
-                
-        uint256 _vaultId;
+        console.log('get amounts to LP amount0Desired %s amount1Desired %s', amount0Desired, amount1Desired);
+        uint256 _vaultId = _mintAndLpParams.vaultId;
```

Comment:
When u rebase this, this change should disappear as the fix is already in main branch